manifests/files.pp

```diff
@@ -43,12 +44,18 @@
   }
 
   # Aliases
-  file { '/etc/aliases':
-    ensure  => 'file',
-    content => "# file managed by puppet\n",
-    notify  => Exec['newaliases'],
-    replace => false,
-    seltype => $postfix::params::aliasesseltype,
+  if $manage_aliases {
```

Comment:
Thanks for switching to `if`. However, this means the `true` and `default` sections need to be removed.

Reply:
I am so sorry. I pushed the wrong commit. It is fixed now.

Reply:
I've pushed the write branch.